webpack/assets/javascripts/react_app/components/common/forms/Select.js

```diff
@@ -9,6 +8,7 @@ import { noop } from '../../../common/helpers';
 import CommonForm from './CommonForm';
 import { STATUS } from '../../../constants';
 import MessageBox from '../MessageBox';
+import { renderOptions } from './SelectHelpers';
```

Comment:
Nice refactor :+1:
Should we maybe extract it to a separate PR?
You could even use the same ticket as we've used for #7700 that introduced those methods.